VB-Application/src/main/java/com/gmail/visualbukkit/blocks/definitions/core/ExprColoredString.java

```diff
@@ -1,18 +1,26 @@
 package com.gmail.visualbukkit.blocks.definitions.core;
 
+import com.gmail.visualbukkit.VisualBukkitApp;
 import com.gmail.visualbukkit.blocks.BlockDefinition;
 import com.gmail.visualbukkit.blocks.ExpressionBlock;
 import com.gmail.visualbukkit.blocks.parameters.ExpressionParameter;
 import com.gmail.visualbukkit.project.BuildInfo;
 import com.gmail.visualbukkit.reflection.ClassInfo;
 
+import java.net.URI;
+
 @BlockDefinition(id = "expr-colored-string", name = "Colored String", description = "A string with '&' color codes")
 public class ExprColoredString extends ExpressionBlock {
 
     public ExprColoredString() {
         addParameter("String", new ExpressionParameter(ClassInfo.of(String.class)));
     }
 
+    @Override
+    public void openJavadocs() {
+        VisualBukkitApp.openURI(URI.create("https://jd.papermc.io/paper/1.21.4/org/bukkit/ChatColor.html"));
```

Comment:
add a constant for the base url "https://jd.papermc.io/paper/1.21.4" or a utility method or something to avoid hardcoding it in a million places